### PR TITLE
Add Felony 0.10.3

### PR DESCRIPTION
**Pull request by brettneese**
Add Felony 0.10.3

https://github.com/henryboldi/felony
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

:)

### CODE REVIEW

File: Casks/felony.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'felony' do
+  version '0.10.3'
+  sha256 'b9587e374683ce6fc520f63ae6559fbda801d80e4d0026380f4144541e6fb715'
+
+  url 'https://github.com/henryboldi/felony/releases/download/0.10.3/Felony-0.10.3-mac.zip'
```
Comment (vitorgalvao) [inline Casks/felony.rb]: Please add a `appcast 'url 'https://github.com/henryboldi/felony/releases/download/0.10.3/Felony-0.10.3-mac.zip'.atom'` line under.

Comment (vitorgalvao) [inline Casks/felony.rb]: `url "https://github.com/henryboldi/felony/releases/download/#{version}/Felony-#{version}-mac.zip"`

Comment (vitorgalvao) [inline Casks/felony.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by brettneese** [comment]
Oops, forgot to check style before committing. One sec. 

Also, I'm getting 
image

when I update the URL? Is that OK?

File: Casks/felony.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'felony' do
+  version '0.10.3'
+  sha256 'b9587e374683ce6fc520f63ae6559fbda801d80e4d0026380f4144541e6fb715'
+
+  url 'https://github.com/henryboldi/felony/releases/download/#{version}/Felony-#{version}-mac.zip'
```
Comment (reitermarkus) [inline Casks/felony.rb]: You need to used double quotes here, otherwise `#{version}` will not be interpolated.

**Comment by brettneese** [comment]
@reitermarkus I did that, but still get 
image

**Comment by reitermarkus** [comment]
Ok, then it's the `appcast`, which is wrong anyway. Take a look at the link @vitorgalvao posted on the diff above.

**Comment by brettneese** [comment]
You're right. That wasn't covered very well in the contributing guide and I got confused. Should be good now?

**Comment by fanquake** [comment]
```
Running script.sh...
>>> developer/bin/audit_modified_casks f03c43ffe2dbbb915c0586f00a82045aa4afc55a...2eded5f2201dabff596ff897976bc5ffc1d5e394
==> 1 modified cask: felony
==> Downloading https://github.com/henryboldi/felony/releases/download/0.10.3/Fe
######################################################################## 100.0%
==> Verifying checksum for Cask felony
audit for felony: warning
 - appcast checkpoint mismatch
Expected: e94cd1ccc01488e99b07a372162ea9e555b6aef0e0911cbb198281f03b975467
Actual: 17e205d3176417046658cb1c3e48d388334ba10c8213faa8fcf0347e49c1084b
Error: audit failed for 1 cask: felony
```

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 5db2c640409295a92ccb183c14d0c777570ddf28. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**